src/perl/Permabit/BlockDevice/VDO/LVMVDO.pm

```diff
@@ -183,40 +162,53 @@ sub installModule {
 }
 
 ########################################################################
-# @inherit
+# Get the lvm name of the device on top of VDO. Implemented in inheriting
+# classes.
 ##
-sub growLogical {
-  my ($self, $logicalSize) = assertNumArgs(2, @_);
```

Comment:
It seem to me that, much like renameVDO(), growPlogical() and growPhysical() should still be defined in the parent class, with an implmentation thta errors out if a subclass fails to override it. (That would also make these diffs cleaner, I think.)